### ISSUE TITLE
Unexpected loss of feature using suitcase - enhancement request?

### ISSUE THREAD

**Issue by KollinsPlays** [bafomdad/realfilingcabinet#90]
Minecraft 1.12.2
Skyfactory 4-4.1.0 modpack
Real Filing Cabinet 1.12-0.2.0.20

Behavior:
When carrying a folder of dirt and one of torches anywhere in inventory (not just on the hotbar or main or offhand), any dirt or torches picked up skip normal inventory and go in their folder.

After crafting a Suitcase and putting the dirt and torch folders inside, drops now only go in the folders if the suitcase is in the main or offhand.

Expected result: 
Upgrading from carrying individual folders to putting them in a suitcase should be more useful than just keeping multiple folders somewhere in inventory, but having to keep the suitcase in hand means giving up a shield or other offhand item vs still filing up your inventory with the blocks you were carrying folders around for in the 1st place.

Oh an another request for the suitcase is that the wiki states, "Suitcases also comes with a tooltip mentioning a slot index" but it's not there. The standard tooltip says "Suitcase" and to get the detailed tooltip, I have to open inventory and SHIFT-mouseover to see the slot index. Can the behavior be changed so the non-detailed tooltip also shows something like Suitcase <contents of focused slot's folder>?

**Comment by bafomdad** [comment]
I agree that the tooltip of the suitcase needed some tweaking. A recent update (0.2.0.21) has made the suitcase's tooltip display a little better. Additionally, there is now a small HUD in the bottom left corner that displays the content of items/blocks currently contained within any folder, including the folder currently indexed within the suitcase.

The suitcase can be considered a very portable filing cabinet, so the design philosophy is to not make the overall mod _too_ convenient. I will see if this can't be a config option later on though
